# Russian message translation file for libpq
# Copyright (C) 2001-2016 PostgreSQL Global Development Group
# This file is distributed under the same license as the PostgreSQL package.
# Serguei A. Mokhov <mokhov@cs.concordia.ca>, 2001-2004.
# Oleg Bartunov <oleg@sai.msu.su>, 2005.
# Andrey Sudnik <sudnikand@yandex.ru>, 2010.
<<<<<<< HEAD
# Alexander Lakhin <exclusion@gmail.com>, 2012-2017, 2018, 2019, 2020, 2021, 2022.
=======
# SPDX-FileCopyrightText: 2012-2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025 Alexander Lakhin <exclusion@gmail.com>
>>>>>>> REL_16_9
# Maxim Yablokov <m.yablokov@postgrespro.ru>, 2021.
msgid ""
msgstr ""
"Project-Id-Version: libpq (PostgreSQL current)\n"
"Report-Msgid-Bugs-To: pgsql-bugs@lists.postgresql.org\n"
<<<<<<< HEAD
"POT-Creation-Date: 2022-05-07 06:06+0300\n"
"PO-Revision-Date: 2022-05-07 06:33+0300\n"
=======
"POT-Creation-Date: 2025-05-03 15:59+0300\n"
"PO-Revision-Date: 2025-05-03 16:34+0300\n"
>>>>>>> REL_16_9
"Last-Translator: Alexander Lakhin <exclusion@gmail.com>\n"
"Language-Team: Russian <pgsql-ru-general@postgresql.org>\n"
"Language: ru\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=3; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && "
"n%10<=4 && (n%100<10 || n%100>=20) ? 1 : 2);\n"

<<<<<<< HEAD
#: fe-auth-scram.c:213
msgid "malformed SCRAM message (empty message)\n"
msgstr "неправильное сообщение SCRAM (пустое содержимое)\n"

#: fe-auth-scram.c:219
msgid "malformed SCRAM message (length mismatch)\n"
msgstr "неправильное сообщение SCRAM (некорректная длина)\n"

#: fe-auth-scram.c:263
msgid "could not verify server signature\n"
msgstr "не удалось проверить сигнатуру сервера\n"

#: fe-auth-scram.c:270
msgid "incorrect server signature\n"
msgstr "некорректная сигнатура сервера\n"

#: fe-auth-scram.c:279
msgid "invalid SCRAM exchange state\n"
msgstr "ошибочное состояние обмена SCRAM\n"

#: fe-auth-scram.c:306
=======
#: ../../port/thread.c:50 ../../port/thread.c:86
>>>>>>> REL_16_9
#, c-format
msgid "could not look up local user ID %d: %s"
msgstr "найти локального пользователя по идентификатору (%d) не удалось: %s"

<<<<<<< HEAD
#: fe-auth-scram.c:315
=======
#: ../../port/thread.c:55 ../../port/thread.c:91
>>>>>>> REL_16_9
#, c-format
msgid "local user with ID %d does not exist"
msgstr "локальный пользователь с ID %d не существует"

#: fe-auth-scram.c:227
#, c-format
msgid "malformed SCRAM message (empty message)"
msgstr "неправильное сообщение SCRAM (пустое содержимое)"

#: fe-auth-scram.c:232
#, c-format
msgid "malformed SCRAM message (length mismatch)"
msgstr "неправильное сообщение SCRAM (некорректная длина)"

#: fe-auth-scram.c:275
#, c-format
msgid "could not verify server signature: %s"
msgstr "не удалось проверить сигнатуру сервера: %s"

#: fe-auth-scram.c:281
#, c-format
msgid "incorrect server signature"
msgstr "некорректная сигнатура сервера"

#: fe-auth-scram.c:290
#, c-format
msgid "invalid SCRAM exchange state"
msgstr "ошибочное состояние обмена SCRAM"

#: fe-auth-scram.c:317
#, c-format
msgid "malformed SCRAM message (attribute \"%c\" expected)"
msgstr "неправильное сообщение SCRAM (ожидался атрибут \"%c\")"

#: fe-auth-scram.c:326
#, c-format
msgid "malformed SCRAM message (expected character \"=\" for attribute \"%c\")"
msgstr ""
"неправильное сообщение SCRAM (для атрибута \"%c\" ожидался символ \"=\")"

<<<<<<< HEAD
#: fe-auth-scram.c:356
msgid "could not generate nonce\n"
msgstr "не удалось сгенерировать разовый код\n"

#: fe-auth-scram.c:366 fe-auth-scram.c:441 fe-auth-scram.c:595
#: fe-auth-scram.c:616 fe-auth-scram.c:642 fe-auth-scram.c:657
#: fe-auth-scram.c:707 fe-auth-scram.c:746 fe-auth.c:290 fe-auth.c:362
#: fe-auth.c:398 fe-auth.c:615 fe-auth.c:774 fe-auth.c:1132 fe-auth.c:1282
#: fe-connect.c:911 fe-connect.c:1460 fe-connect.c:1629 fe-connect.c:2981
#: fe-connect.c:4711 fe-connect.c:4972 fe-connect.c:5091 fe-connect.c:5343
#: fe-connect.c:5424 fe-connect.c:5523 fe-connect.c:5779 fe-connect.c:5808
#: fe-connect.c:5880 fe-connect.c:5904 fe-connect.c:5922 fe-connect.c:6023
#: fe-connect.c:6032 fe-connect.c:6390 fe-connect.c:6540 fe-connect.c:6806
#: fe-exec.c:686 fe-exec.c:876 fe-exec.c:1223 fe-exec.c:3047 fe-exec.c:3230
#: fe-exec.c:4003 fe-exec.c:4168 fe-gssapi-common.c:111 fe-lobj.c:881
#: fe-protocol3.c:975 fe-protocol3.c:990 fe-protocol3.c:1023
#: fe-protocol3.c:1731 fe-secure-common.c:110 fe-secure-gssapi.c:504
#: fe-secure-openssl.c:440 fe-secure-openssl.c:1133
msgid "out of memory\n"
msgstr "нехватка памяти\n"

#: fe-auth-scram.c:374
msgid "could not encode nonce\n"
msgstr "не удалось оформить разовый код\n"

#: fe-auth-scram.c:563
msgid "could not calculate client proof\n"
msgstr "не удалось вычислить подтверждение клиента\n"

#: fe-auth-scram.c:579
msgid "could not encode client proof\n"
msgstr "не удалось закодировать подтверждение клиента\n"

#: fe-auth-scram.c:634
msgid "invalid SCRAM response (nonce mismatch)\n"
msgstr "неверный ответ SCRAM (несовпадение проверочного кода)\n"

#: fe-auth-scram.c:667
msgid "malformed SCRAM message (invalid salt)\n"
msgstr "неправильное сообщение SCRAM (некорректная соль)\n"

#: fe-auth-scram.c:681
msgid "malformed SCRAM message (invalid iteration count)\n"
msgstr "неправильное сообщение SCRAM (некорректное число итераций)\n"

#: fe-auth-scram.c:687
msgid "malformed SCRAM message (garbage at end of server-first-message)\n"
msgstr ""
"неправильное сообщение SCRAM (мусор в конце первого сообщения сервера)\n"

#: fe-auth-scram.c:723
=======
#: fe-auth-scram.c:366
>>>>>>> REL_16_9
#, c-format
msgid "could not generate nonce"
msgstr "не удалось сгенерировать разовый код"

<<<<<<< HEAD
#: fe-auth-scram.c:739
msgid "malformed SCRAM message (garbage at end of server-final-message)\n"
msgstr ""
"неправильное сообщение SCRAM (мусор в конце последнего сообщения сервера)\n"

#: fe-auth-scram.c:758
msgid "malformed SCRAM message (invalid server signature)\n"
msgstr "неправильное сообщение SCRAM (неверная сигнатура сервера)\n"

#: fe-auth.c:76
=======
#: fe-auth-scram.c:375 fe-auth-scram.c:448 fe-auth-scram.c:600
#: fe-auth-scram.c:620 fe-auth-scram.c:644 fe-auth-scram.c:658
#: fe-auth-scram.c:704 fe-auth-scram.c:740 fe-auth-scram.c:914 fe-auth.c:296
#: fe-auth.c:369 fe-auth.c:403 fe-auth.c:618 fe-auth.c:729 fe-auth.c:1210
#: fe-auth.c:1375 fe-connect.c:925 fe-connect.c:1759 fe-connect.c:1921
#: fe-connect.c:3291 fe-connect.c:4492 fe-connect.c:5157 fe-connect.c:5412
#: fe-connect.c:5530 fe-connect.c:5777 fe-connect.c:5857 fe-connect.c:5955
#: fe-connect.c:6206 fe-connect.c:6233 fe-connect.c:6309 fe-connect.c:6332
#: fe-connect.c:6356 fe-connect.c:6391 fe-connect.c:6477 fe-connect.c:6485
#: fe-connect.c:6842 fe-connect.c:6992 fe-exec.c:527 fe-exec.c:1323
#: fe-exec.c:3132 fe-exec.c:4157 fe-exec.c:4323 fe-gssapi-common.c:109
#: fe-lobj.c:870 fe-protocol3.c:204 fe-protocol3.c:228 fe-protocol3.c:251
#: fe-protocol3.c:268 fe-protocol3.c:348 fe-protocol3.c:715 fe-protocol3.c:954
#: fe-protocol3.c:1765 fe-protocol3.c:2165 fe-secure-common.c:110
#: fe-secure-gssapi.c:496 fe-secure-openssl.c:435 fe-secure-openssl.c:1271
>>>>>>> REL_16_9
#, c-format
msgid "out of memory"
msgstr "нехватка памяти"

#: fe-auth-scram.c:382
#, c-format
msgid "could not encode nonce"
msgstr "не удалось оформить разовый код"

#: fe-auth-scram.c:570
#, c-format
msgid "could not calculate client proof: %s"
msgstr "не удалось вычислить подтверждение клиента: %s"

#: fe-auth-scram.c:585
#, c-format
msgid "could not encode client proof"
msgstr "не удалось закодировать подтверждение клиента"

#: fe-auth-scram.c:637
#, c-format
msgid "invalid SCRAM response (nonce mismatch)"
msgstr "неверный ответ SCRAM (несовпадение разового кода)"

#: fe-auth-scram.c:667
#, c-format
msgid "malformed SCRAM message (invalid salt)"
msgstr "неправильное сообщение SCRAM (некорректная соль)"

#: fe-auth-scram.c:680
#, c-format
msgid "malformed SCRAM message (invalid iteration count)"
msgstr "неправильное сообщение SCRAM (некорректное число итераций)"

#: fe-auth-scram.c:685
#, c-format
msgid "malformed SCRAM message (garbage at end of server-first-message)"
msgstr "неправильное сообщение SCRAM (мусор в конце первого сообщения сервера)"

#: fe-auth-scram.c:719
#, c-format
msgid "error received from server in SCRAM exchange: %s"
msgstr "в ходе обмена SCRAM от сервера получена ошибка: %s"

#: fe-auth-scram.c:734
#, c-format
msgid "malformed SCRAM message (garbage at end of server-final-message)"
msgstr ""
"неправильное сообщение SCRAM (мусор в конце последнего сообщения сервера)"

#: fe-auth-scram.c:751
#, c-format
msgid "malformed SCRAM message (invalid server signature)"
msgstr "неправильное сообщение SCRAM (неверная сигнатура сервера)"

#: fe-auth-scram.c:923
msgid "could not generate random salt"
msgstr "не удалось сгенерировать случайную соль"

#: fe-auth.c:77
#, c-format
msgid "out of memory allocating GSSAPI buffer (%d)"
msgstr "недостаточно памяти для буфера GSSAPI (%d)"

#: fe-auth.c:138
msgid "GSSAPI continuation error"
msgstr "ошибка продолжения в GSSAPI"

<<<<<<< HEAD
#: fe-auth.c:158 fe-auth.c:391 fe-gssapi-common.c:98 fe-secure-common.c:98
msgid "host name must be specified\n"
msgstr "требуется указать имя сервера\n"

#: fe-auth.c:165
msgid "duplicate GSS authentication request\n"
msgstr "повторный запрос аутентификации GSS\n"

#: fe-auth.c:230
=======
#: fe-auth.c:168 fe-auth.c:397 fe-gssapi-common.c:97 fe-secure-common.c:99
#: fe-secure-common.c:173
>>>>>>> REL_16_9
#, c-format
msgid "host name must be specified"
msgstr "требуется указать имя сервера"

#: fe-auth.c:174
#, c-format
msgid "duplicate GSS authentication request"
msgstr "повторный запрос аутентификации GSS"

#: fe-auth.c:238
#, c-format
msgid "out of memory allocating SSPI buffer (%d)"
msgstr "недостаточно памяти для буфера SSPI (%d)"

#: fe-auth.c:285
msgid "SSPI continuation error"
msgstr "ошибка продолжения в SSPI"

<<<<<<< HEAD
#: fe-auth.c:351
msgid "duplicate SSPI authentication request\n"
msgstr "повторный запрос аутентификации SSPI\n"

#: fe-auth.c:377
msgid "could not acquire SSPI credentials"
msgstr "не удалось получить удостоверение SSPI"

#: fe-auth.c:433
msgid "channel binding required, but SSL not in use\n"
msgstr "требуется привязка каналов, но SSL не используется\n"

#: fe-auth.c:440
msgid "duplicate SASL authentication request\n"
msgstr "повторный запрос аутентификации SASL\n"

#: fe-auth.c:496
msgid "channel binding is required, but client does not support it\n"
msgstr "требуется привязка каналов, но клиент её не поддерживает\n"

#: fe-auth.c:513
=======
#: fe-auth.c:359
#, c-format
msgid "duplicate SSPI authentication request"
msgstr "повторный запрос аутентификации SSPI"

#: fe-auth.c:384
msgid "could not acquire SSPI credentials"
msgstr "не удалось получить удостоверение SSPI"

#: fe-auth.c:437
#, c-format
msgid "channel binding required, but SSL not in use"
msgstr "требуется привязка каналов, но SSL не используется"

#: fe-auth.c:443
#, c-format
msgid "duplicate SASL authentication request"
msgstr "повторный запрос аутентификации SASL"

#: fe-auth.c:501
#, c-format
msgid "channel binding is required, but client does not support it"
msgstr "требуется привязка каналов, но клиент её не поддерживает"

#: fe-auth.c:517
#, c-format
>>>>>>> REL_16_9
msgid ""
"server offered SCRAM-SHA-256-PLUS authentication over a non-SSL connection"
msgstr ""
"сервер предложил аутентификацию SCRAM-SHA-256-PLUS для соединения, не "
"защищённого SSL"

<<<<<<< HEAD
#: fe-auth.c:525
msgid "none of the server's SASL authentication mechanisms are supported\n"
msgstr ""
"ни один из серверных механизмов аутентификации SASL не поддерживается\n"

#: fe-auth.c:533
=======
#: fe-auth.c:531
#, c-format
msgid "none of the server's SASL authentication mechanisms are supported"
msgstr "ни один из серверных механизмов аутентификации SASL не поддерживается"

#: fe-auth.c:538
#, c-format
>>>>>>> REL_16_9
msgid ""
"channel binding is required, but server did not offer an authentication "
"method that supports channel binding"
msgstr ""
"требуется привязка каналов, но сервер не предложил поддерживающий её метод "
"аутентификации"

<<<<<<< HEAD
#: fe-auth.c:639
=======
#: fe-auth.c:641
>>>>>>> REL_16_9
#, c-format
msgid "out of memory allocating SASL buffer (%d)"
msgstr "недостаточно памяти для буфера SASL (%d)"

<<<<<<< HEAD
#: fe-auth.c:664
=======
#: fe-auth.c:665
#, c-format
>>>>>>> REL_16_9
msgid ""
"AuthenticationSASLFinal received from server, but SASL authentication was "
"not completed"
msgstr ""
"c сервера получено сообщение AuthenticationSASLFinal, но аутентификация SASL "
"ещё не завершена"

<<<<<<< HEAD
#: fe-auth.c:741
msgid "SCM_CRED authentication method not supported\n"
msgstr "аутентификация SCM_CRED не поддерживается\n"
=======
#: fe-auth.c:675
#, c-format
msgid "no client response found after SASL exchange success"
msgstr "после успешного обмена по протоколу SASL не получен ответ клиента"
>>>>>>> REL_16_9

#: fe-auth.c:738 fe-auth.c:745 fe-auth.c:1358 fe-auth.c:1369
#, c-format
msgid "could not encrypt password: %s"
msgstr "не удалось зашифровать пароль: %s"

#: fe-auth.c:773
msgid "server requested a cleartext password"
msgstr "сервер запросил незашифрованный пароль"

#: fe-auth.c:775
msgid "server requested a hashed password"
msgstr "сервер запросил хешированный пароль"

#: fe-auth.c:778
msgid "server requested GSSAPI authentication"
msgstr "сервер запросил аутентификацию GSSAPI"

#: fe-auth.c:780
msgid "server requested SSPI authentication"
msgstr "сервер запросил аутентификацию SSPI"

#: fe-auth.c:784
msgid "server requested SASL authentication"
msgstr "сервер запросил аутентификацию SASL"

#: fe-auth.c:787
msgid "server requested an unknown authentication type"
msgstr "сервер запросил аутентификацию неизвестного типа"

#: fe-auth.c:820
#, c-format
msgid "server did not request an SSL certificate"
msgstr "сервер не запросил сертификат SSL"

#: fe-auth.c:825
#, c-format
msgid "server accepted connection without a valid SSL certificate"
msgstr "сервер принял подключение, не проверив сертификат SSL"

#: fe-auth.c:879
msgid "server did not complete authentication"
msgstr "сервер не завершил аутентификацию"

#: fe-auth.c:913
#, c-format
msgid "authentication method requirement \"%s\" failed: %s"
msgstr "требование метода аутентификации \"%s\" не выполнено: %s"

#: fe-auth.c:936
#, c-format
msgid ""
"channel binding required, but server authenticated client without channel "
"binding"
msgstr ""
"требуется привязка каналов, но сервер аутентифицировал клиента без привязки"

#: fe-auth.c:941
#, c-format
msgid ""
"channel binding required but not supported by server's authentication request"
msgstr ""
"требуется привязка каналов, но она не поддерживается при том запросе "
"аутентификации, который передал сервер"

<<<<<<< HEAD
#: fe-auth.c:877
msgid "Kerberos 4 authentication not supported\n"
msgstr "аутентификация Kerberos 4 не поддерживается\n"

#: fe-auth.c:882
msgid "Kerberos 5 authentication not supported\n"
msgstr "аутентификация Kerberos 5 не поддерживается\n"

#: fe-auth.c:953
msgid "GSSAPI authentication not supported\n"
msgstr "аутентификация через GSSAPI не поддерживается\n"

#: fe-auth.c:985
msgid "SSPI authentication not supported\n"
msgstr "аутентификация через SSPI не поддерживается\n"

#: fe-auth.c:993
msgid "Crypt authentication not supported\n"
msgstr "аутентификация Crypt не поддерживается\n"

#: fe-auth.c:1060
=======
#: fe-auth.c:975
>>>>>>> REL_16_9
#, c-format
msgid "Kerberos 4 authentication not supported"
msgstr "аутентификация Kerberos 4 не поддерживается"

<<<<<<< HEAD
#: fe-auth.c:1107
=======
#: fe-auth.c:979
>>>>>>> REL_16_9
#, c-format
msgid "Kerberos 5 authentication not supported"
msgstr "аутентификация Kerberos 5 не поддерживается"

<<<<<<< HEAD
#: fe-auth.c:1117 fe-connect.c:2856
=======
#: fe-auth.c:1049
>>>>>>> REL_16_9
#, c-format
msgid "GSSAPI authentication not supported"
msgstr "аутентификация через GSSAPI не поддерживается"

<<<<<<< HEAD
#: fe-auth.c:1122 fe-connect.c:2861
=======
#: fe-auth.c:1080
>>>>>>> REL_16_9
#, c-format
msgid "SSPI authentication not supported"
msgstr "аутентификация через SSPI не поддерживается"

<<<<<<< HEAD
#: fe-auth.c:1226
msgid "unexpected shape of result set returned for SHOW\n"
msgstr "неожиданная форма набора результатов, возвращённого для SHOW\n"

#: fe-auth.c:1235
msgid "password_encryption value too long\n"
msgstr "слишком длинное значение password_encryption\n"

#: fe-auth.c:1275
=======
#: fe-auth.c:1087
>>>>>>> REL_16_9
#, c-format
msgid "Crypt authentication not supported"
msgstr "аутентификация Crypt не поддерживается"

<<<<<<< HEAD
#: fe-connect.c:1094
=======
#: fe-auth.c:1151
>>>>>>> REL_16_9
#, c-format
msgid "authentication method %u not supported"
msgstr "метод аутентификации %u не поддерживается"

<<<<<<< HEAD
#: fe-connect.c:1180
=======
#: fe-auth.c:1197
>>>>>>> REL_16_9
#, c-format
msgid "user name lookup failure: error code %lu"
msgstr "распознать имя пользователя не удалось (код ошибки: %lu)"

<<<<<<< HEAD
#: fe-connect.c:1273 fe-connect.c:1299 fe-connect.c:1341 fe-connect.c:1350
#: fe-connect.c:1383 fe-connect.c:1427
#, c-format
msgid "invalid %s value: \"%s\"\n"
msgstr "неверное значение %s: \"%s\"\n"

#: fe-connect.c:1320
=======
#: fe-auth.c:1321
#, c-format
msgid "unexpected shape of result set returned for SHOW"
msgstr "неожиданная форма набора результатов, возвращённого для SHOW"

#: fe-auth.c:1329
#, c-format
msgid "password_encryption value too long"
msgstr "слишком длинное значение password_encryption"

#: fe-auth.c:1379
>>>>>>> REL_16_9
#, c-format
msgid "unrecognized password encryption algorithm \"%s\""
msgstr "нераспознанный алгоритм шифрования пароля \"%s\""

<<<<<<< HEAD
#: fe-connect.c:1368
msgid "invalid SSL protocol version range\n"
msgstr "неверный диапазон версий протокола SSL\n"

#: fe-connect.c:1393
=======
#: fe-connect.c:1132
#, c-format
msgid "could not match %d host names to %d hostaddr values"
msgstr "не удалось сопоставить имена узлов (%d) со значениями hostaddr (%d)"

#: fe-connect.c:1212
#, c-format
msgid "could not match %d port numbers to %d hosts"
msgstr "не удалось сопоставить номера портов (%d) с узлами (%d)"

#: fe-connect.c:1337
>>>>>>> REL_16_9
#, c-format
msgid ""
"negative require_auth method \"%s\" cannot be mixed with non-negative methods"
msgstr ""
"отрицательный метод require_auth \"%s\" не может совмещаться с "
"неотрицательными методами"

<<<<<<< HEAD
#: fe-connect.c:1653
=======
#: fe-connect.c:1350
#, c-format
msgid "require_auth method \"%s\" cannot be mixed with negative methods"
msgstr ""
"метод require_auth \"%s\" не может совмещаться с отрицательными методами"

#: fe-connect.c:1410 fe-connect.c:1461 fe-connect.c:1503 fe-connect.c:1559
#: fe-connect.c:1567 fe-connect.c:1598 fe-connect.c:1644 fe-connect.c:1684
#: fe-connect.c:1705
>>>>>>> REL_16_9
#, c-format
msgid "invalid %s value: \"%s\""
msgstr "неверное значение %s: \"%s\""

<<<<<<< HEAD
#: fe-connect.c:1715
#, c-format
msgid "connection to server on socket \"%s\" failed: "
msgstr "подключиться к серверу через сокет \"%s\" не удалось: "

#: fe-connect.c:1742
#, c-format
msgid "connection to server at \"%s\" (%s), port %s failed: "
msgstr "подключиться к серверу \"%s\" (%s), порту %s не удалось: "

#: fe-connect.c:1747
#, c-format
msgid "connection to server at \"%s\", port %s failed: "
msgstr "подключиться к серверу \"%s\", порту %s не удалось: "

#: fe-connect.c:1772
msgid ""
"\tIs the server running locally and accepting connections on that socket?\n"
msgstr ""
"\tСервер действительно работает локально и принимает подключения через этот "
"сокет?\n"

#: fe-connect.c:1776
msgid ""
"\tIs the server running on that host and accepting TCP/IP connections?\n"
msgstr ""
"\tСервер действительно работает по данному адресу и принимает TCP-"
"соединения?\n"

#: fe-connect.c:1840
=======
#: fe-connect.c:1443
#, c-format
msgid "require_auth method \"%s\" is specified more than once"
msgstr "метод require_auth \"%s\" указан неоднократно"

#: fe-connect.c:1484 fe-connect.c:1523 fe-connect.c:1606
#, c-format
msgid "%s value \"%s\" invalid when SSL support is not compiled in"
msgstr "значение %s \"%s\" недопустимо для сборки без поддержки SSL"

#: fe-connect.c:1546
#, c-format
msgid ""
"weak sslmode \"%s\" may not be used with sslrootcert=system (use \"verify-"
"full\")"
msgstr ""
"слабый режим sslmode \"%s\" не может использоваться с sslrootcert=system "
"(используйте режим \"verify-full\")"

#: fe-connect.c:1584
#, c-format
msgid "invalid SSL protocol version range"
msgstr "неверный диапазон версий протокола SSL"

#: fe-connect.c:1621
#, c-format
msgid "%s value \"%s\" is not supported (check OpenSSL version)"
msgstr "значение %s \"%s\" не поддерживается (проверьте версию OpenSSL)"

#: fe-connect.c:1651
#, c-format
msgid "gssencmode value \"%s\" invalid when GSSAPI support is not compiled in"
msgstr "значение gssencmode \"%s\" недопустимо для сборки без поддержки GSSAPI"

#: fe-connect.c:1944
#, c-format
msgid "could not set socket to TCP no delay mode: %s"
msgstr "не удалось перевести сокет в режим TCP-передачи без задержки: %s"

#: fe-connect.c:2003
#, c-format
msgid "connection to server on socket \"%s\" failed: "
msgstr "подключиться к серверу через сокет \"%s\" не удалось: "

#: fe-connect.c:2029
#, c-format
msgid "connection to server at \"%s\" (%s), port %s failed: "
msgstr "подключиться к серверу \"%s\" (%s), порту %s не удалось: "

#: fe-connect.c:2034
#, c-format
msgid "connection to server at \"%s\", port %s failed: "
msgstr "подключиться к серверу \"%s\", порту %s не удалось: "

#: fe-connect.c:2057
#, c-format
msgid ""
"\tIs the server running locally and accepting connections on that socket?"
msgstr ""
"\tСервер действительно работает локально и принимает подключения через этот "
"сокет?"

#: fe-connect.c:2059
#, c-format
msgid "\tIs the server running on that host and accepting TCP/IP connections?"
msgstr ""
"\tСервер действительно работает по данному адресу и принимает TCP-соединения?"

#: fe-connect.c:2122
>>>>>>> REL_16_9
#, c-format
msgid "invalid integer value \"%s\" for connection option \"%s\""
msgstr "неверное целочисленное значение \"%s\" для параметра соединения \"%s\""

<<<<<<< HEAD
#: fe-connect.c:1870 fe-connect.c:1905 fe-connect.c:1941 fe-connect.c:2030
#: fe-connect.c:2644
#, c-format
msgid "%s(%s) failed: %s\n"
msgstr "ошибка в %s(%s): %s\n"

#: fe-connect.c:1995
#, c-format
msgid "%s(%s) failed: error code %d\n"
msgstr "ошибка в %s(%s): код ошибки %d\n"

#: fe-connect.c:2310
msgid "invalid connection state, probably indicative of memory corruption\n"
msgstr "неверное состояние соединения - возможно разрушение памяти\n"

#: fe-connect.c:2389
=======
#: fe-connect.c:2151 fe-connect.c:2185 fe-connect.c:2220 fe-connect.c:2318
#: fe-connect.c:2973
#, c-format
msgid "%s(%s) failed: %s"
msgstr "ошибка в %s(%s): %s"

#: fe-connect.c:2284
#, c-format
msgid "%s(%s) failed: error code %d"
msgstr "ошибка в %s(%s): код ошибки %d"

#: fe-connect.c:2597
>>>>>>> REL_16_9
#, c-format
msgid "invalid connection state, probably indicative of memory corruption"
msgstr "неверное состояние соединения - возможно разрушение памяти"

<<<<<<< HEAD
#: fe-connect.c:2405
=======
#: fe-connect.c:2676
>>>>>>> REL_16_9
#, c-format
msgid "invalid port number: \"%s\""
msgstr "неверный номер порта: \"%s\""

<<<<<<< HEAD
#: fe-connect.c:2418
=======
#: fe-connect.c:2690
>>>>>>> REL_16_9
#, c-format
msgid "could not translate host name \"%s\" to address: %s"
msgstr "преобразовать имя \"%s\" в адрес не удалось: %s"

<<<<<<< HEAD
#: fe-connect.c:2431
=======
#: fe-connect.c:2702
>>>>>>> REL_16_9
#, c-format
msgid "could not parse network address \"%s\": %s"
msgstr "не удалось разобрать сетевой адрес \"%s\": %s"

<<<<<<< HEAD
#: fe-connect.c:2446
=======
#: fe-connect.c:2713
>>>>>>> REL_16_9
#, c-format
msgid "Unix-domain socket path \"%s\" is too long (maximum %d bytes)"
msgstr "длина пути Unix-сокета \"%s\" превышает предел (%d байт)"

<<<<<<< HEAD
#: fe-connect.c:2572
=======
#: fe-connect.c:2727
>>>>>>> REL_16_9
#, c-format
msgid "could not translate Unix-domain socket path \"%s\" to address: %s"
msgstr "преобразовать путь Unix-сокета \"%s\" в адрес не удалось: %s"

<<<<<<< HEAD
#: fe-connect.c:2603
=======
#: fe-connect.c:2901
>>>>>>> REL_16_9
#, c-format
msgid "could not create socket: %s"
msgstr "не удалось создать сокет: %s"

<<<<<<< HEAD
#: fe-connect.c:2613
=======
#: fe-connect.c:2932
>>>>>>> REL_16_9
#, c-format
msgid "could not set socket to nonblocking mode: %s"
msgstr "не удалось перевести сокет в неблокирующий режим: %s"

#: fe-connect.c:2943
#, c-format
msgid "could not set socket to close-on-exec mode: %s"
msgstr ""
"не удалось перевести сокет в режим закрытия при выполнении (close-on-exec): "
"%s"

<<<<<<< HEAD
#: fe-connect.c:2631
msgid "keepalives parameter must be an integer\n"
msgstr "параметр keepalives должен быть целым числом\n"

#: fe-connect.c:2772
=======
#: fe-connect.c:3100
>>>>>>> REL_16_9
#, c-format
msgid "could not get socket error status: %s"
msgstr "не удалось получить статус ошибки сокета: %s"

<<<<<<< HEAD
#: fe-connect.c:2800
=======
#: fe-connect.c:3127
>>>>>>> REL_16_9
#, c-format
msgid "could not get client address from socket: %s"
msgstr "не удалось получить адрес клиента из сокета: %s"

<<<<<<< HEAD
#: fe-connect.c:2842
msgid "requirepeer parameter is not supported on this platform\n"
msgstr "параметр requirepeer не поддерживается в этой ОС\n"

#: fe-connect.c:2845
=======
#: fe-connect.c:3165
>>>>>>> REL_16_9
#, c-format
msgid "requirepeer parameter is not supported on this platform"
msgstr "параметр requirepeer не поддерживается в этой ОС"

<<<<<<< HEAD
#: fe-connect.c:2869
=======
#: fe-connect.c:3167
>>>>>>> REL_16_9
#, c-format
msgid "could not get peer credentials: %s"
msgstr "не удалось получить учётные данные сервера: %s"

#: fe-connect.c:3180
#, c-format
msgid "requirepeer specifies \"%s\", but actual peer user name is \"%s\""
msgstr ""
"requirepeer допускает подключение только к \"%s\", но сервер работает под "
"именем \"%s\""

<<<<<<< HEAD
#: fe-connect.c:2909
=======
#: fe-connect.c:3221
>>>>>>> REL_16_9
#, c-format
msgid "could not send GSSAPI negotiation packet: %s"
msgstr "не удалось отправить пакет согласования GSSAPI: %s"

<<<<<<< HEAD
#: fe-connect.c:2921
=======
#: fe-connect.c:3233
#, c-format
>>>>>>> REL_16_9
msgid ""
"GSSAPI encryption required but was impossible (possibly no credential cache, "
"no server support, or using a local socket)"
msgstr ""
"затребовано шифрование GSSAPI, но это требование невыполнимо (возможно, "
"отсутствует кеш учётных данных, нет поддержки на сервере или используется "
"локальный сокет)"

<<<<<<< HEAD
#: fe-connect.c:2963
=======
#: fe-connect.c:3274
>>>>>>> REL_16_9
#, c-format
msgid "could not send SSL negotiation packet: %s"
msgstr "не удалось отправить пакет согласования SSL: %s"

<<<<<<< HEAD
#: fe-connect.c:2994
=======
#: fe-connect.c:3303
>>>>>>> REL_16_9
#, c-format
msgid "could not send startup packet: %s"
msgstr "не удалось отправить стартовый пакет: %s"

<<<<<<< HEAD
#: fe-connect.c:3070
msgid "server does not support SSL, but SSL was required\n"
msgstr "затребовано подключение через SSL, но сервер не поддерживает SSL\n"

#: fe-connect.c:3097
=======
#: fe-connect.c:3378
>>>>>>> REL_16_9
#, c-format
msgid "server does not support SSL, but SSL was required"
msgstr "затребовано подключение через SSL, но сервер не поддерживает SSL"

<<<<<<< HEAD
#: fe-connect.c:3118
msgid "received unencrypted data after SSL response\n"
msgstr "после ответа SSL получены незашифрованные данные\n"

#: fe-connect.c:3199
msgid "server doesn't support GSSAPI encryption, but it was required\n"
msgstr "затребовано шифрование GSSAPI, но сервер его не поддерживает\n"

#: fe-connect.c:3211
=======
#: fe-connect.c:3395
>>>>>>> REL_16_9
#, c-format
msgid "server sent an error response during SSL exchange"
msgstr "сервер передал ошибочный ответ во время обмена сообщениями SSL"

<<<<<<< HEAD
#: fe-connect.c:3230
msgid "received unencrypted data after GSSAPI encryption response\n"
msgstr ""
"после ответа на запрос шифрования GSSAPI получены незашифрованные данные\n"

#: fe-connect.c:3290 fe-connect.c:3315
=======
#: fe-connect.c:3400
>>>>>>> REL_16_9
#, c-format
msgid "received invalid response to SSL negotiation: %c"
msgstr "получен неверный ответ при согласовании SSL: %c"

<<<<<<< HEAD
#: fe-connect.c:3522
msgid "unexpected message from server during startup\n"
msgstr "неожиданное сообщение от сервера в начале работы\n"

#: fe-connect.c:3614
msgid "session is read-only\n"
msgstr "сеанс не допускает запись\n"

#: fe-connect.c:3617
msgid "session is not read-only\n"
msgstr "сеанс допускает запись\n"

#: fe-connect.c:3671
msgid "server is in hot standby mode\n"
msgstr "сервер работает в режиме горячего резерва\n"

#: fe-connect.c:3674
msgid "server is not in hot standby mode\n"
msgstr "сервер работает не в режиме горячего резерва\n"

#: fe-connect.c:3792 fe-connect.c:3844
#, c-format
msgid "\"%s\" failed\n"
msgstr "выполнить \"%s\" не удалось\n"

#: fe-connect.c:3858
=======
#: fe-connect.c:3420
#, c-format
msgid "received unencrypted data after SSL response"
msgstr "после ответа SSL получены незашифрованные данные"

#: fe-connect.c:3500
#, c-format
msgid "server doesn't support GSSAPI encryption, but it was required"
msgstr "затребовано шифрование GSSAPI, но сервер его не поддерживает"

#: fe-connect.c:3511
#, c-format
msgid "received invalid response to GSSAPI negotiation: %c"
msgstr "получен неверный ответ при согласовании GSSAPI: %c"

#: fe-connect.c:3529
#, c-format
msgid "received unencrypted data after GSSAPI encryption response"
msgstr ""
"после ответа на запрос шифрования GSSAPI получены незашифрованные данные"

#: fe-connect.c:3594
#, c-format
msgid "expected authentication request from server, but received %c"
msgstr "ожидался запрос аутентификации от сервера, но получено: %c"

#: fe-connect.c:3621 fe-connect.c:3790
#, c-format
msgid "received invalid authentication request"
msgstr "получен некорректный запрос аутентификации"

#: fe-connect.c:3626 fe-connect.c:3775
#, c-format
msgid "received invalid protocol negotiation message"
msgstr "получено некорректное сообщение согласования протокола"

#: fe-connect.c:3644 fe-connect.c:3698
#, c-format
msgid "received invalid error message"
msgstr "получено некорректное сообщение об ошибке"

#: fe-connect.c:3861
#, c-format
msgid "unexpected message from server during startup"
msgstr "неожиданное сообщение от сервера в начале работы"

#: fe-connect.c:3952
#, c-format
msgid "session is read-only"
msgstr "сеанс не допускает запись"

#: fe-connect.c:3954
#, c-format
msgid "session is not read-only"
msgstr "сеанс допускает запись"

#: fe-connect.c:4007
#, c-format
msgid "server is in hot standby mode"
msgstr "сервер работает в режиме горячего резерва"

#: fe-connect.c:4009
#, c-format
msgid "server is not in hot standby mode"
msgstr "сервер работает не в режиме горячего резерва"

#: fe-connect.c:4125 fe-connect.c:4175
#, c-format
msgid "\"%s\" failed"
msgstr "выполнить \"%s\" не удалось"

#: fe-connect.c:4189
#, c-format
msgid "invalid connection state %d, probably indicative of memory corruption"
msgstr "неверное состояние соединения %d - возможно разрушение памяти"

#: fe-connect.c:5170
#, c-format
msgid "invalid LDAP URL \"%s\": scheme must be ldap://"
msgstr "некорректный адрес LDAP \"%s\": схема должна быть ldap://"

#: fe-connect.c:5185
#, c-format
msgid "invalid LDAP URL \"%s\": missing distinguished name"
msgstr "некорректный адрес LDAP \"%s\": отсутствует уникальное имя"

#: fe-connect.c:5197 fe-connect.c:5255
#, c-format
msgid "invalid LDAP URL \"%s\": must have exactly one attribute"
msgstr "некорректный адрес LDAP \"%s\": должен быть только один атрибут"

#: fe-connect.c:5209 fe-connect.c:5271
#, c-format
msgid "invalid LDAP URL \"%s\": must have search scope (base/one/sub)"
msgstr ""
"некорректный адрес LDAP \"%s\": не указана область поиска (base/one/sub)"

#: fe-connect.c:5221
>>>>>>> REL_16_9
#, c-format
msgid "invalid LDAP URL \"%s\": no filter"
msgstr "некорректный адрес LDAP \"%s\": нет фильтра"

<<<<<<< HEAD
#: fe-connect.c:4304 fe-connect.c:4364
=======
#: fe-connect.c:5243
>>>>>>> REL_16_9
#, c-format
msgid "invalid LDAP URL \"%s\": invalid port number"
msgstr "некорректный адрес LDAP \"%s\": неверный номер порта"

<<<<<<< HEAD
#: fe-connect.c:4724
=======
#: fe-connect.c:5280
>>>>>>> REL_16_9
#, c-format
msgid "could not create LDAP structure"
msgstr "не удалось создать структуру LDAP"

<<<<<<< HEAD
#: fe-connect.c:4739
=======
#: fe-connect.c:5355
>>>>>>> REL_16_9
#, c-format
msgid "lookup on LDAP server failed: %s"
msgstr "ошибка поиска на сервере LDAP: %s"

<<<<<<< HEAD
#: fe-connect.c:4751 fe-connect.c:4809
=======
#: fe-connect.c:5365
>>>>>>> REL_16_9
#, c-format
msgid "more than one entry found on LDAP lookup"
msgstr "при поиске LDAP найдено более одного вхождения"

<<<<<<< HEAD
#: fe-connect.c:4763 fe-connect.c:4825
=======
#: fe-connect.c:5367 fe-connect.c:5378
>>>>>>> REL_16_9
#, c-format
msgid "no entry found on LDAP lookup"
msgstr "при поиске LDAP ничего не найдено"

<<<<<<< HEAD
#: fe-connect.c:4775
=======
#: fe-connect.c:5388 fe-connect.c:5400
>>>>>>> REL_16_9
#, c-format
msgid "attribute has no values on LDAP lookup"
msgstr "атрибут не содержит значений при поиске LDAP"

<<<<<<< HEAD
#: fe-connect.c:4797
=======
#: fe-connect.c:5451 fe-connect.c:5470 fe-connect.c:5994
>>>>>>> REL_16_9
#, c-format
msgid "missing \"=\" after \"%s\" in connection info string"
msgstr "в строке соединения нет \"=\" после \"%s\""

<<<<<<< HEAD
#: fe-connect.c:4835
msgid "could not create LDAP structure\n"
msgstr "не удалось создать структуру LDAP\n"

#: fe-connect.c:4911
=======
#: fe-connect.c:5541 fe-connect.c:6177 fe-connect.c:6975
>>>>>>> REL_16_9
#, c-format
msgid "invalid connection option \"%s\""
msgstr "неверный параметр соединения \"%s\""

<<<<<<< HEAD
#: fe-connect.c:4922
msgid "more than one entry found on LDAP lookup\n"
msgstr "при поиске LDAP найдено более одного вхождения\n"

#: fe-connect.c:4923 fe-connect.c:4935
msgid "no entry found on LDAP lookup\n"
msgstr "при поиске LDAP ничего не найдено\n"

#: fe-connect.c:4946 fe-connect.c:4959
msgid "attribute has no values on LDAP lookup\n"
msgstr "атрибут не содержит значений при поиске LDAP\n"

#: fe-connect.c:5011 fe-connect.c:5030 fe-connect.c:5562
=======
#: fe-connect.c:5556 fe-connect.c:6042
>>>>>>> REL_16_9
#, c-format
msgid "unterminated quoted string in connection info string"
msgstr "в строке соединения не хватает закрывающей кавычки"

<<<<<<< HEAD
#: fe-connect.c:5103 fe-connect.c:5747 fe-connect.c:6523
=======
#: fe-connect.c:5636
>>>>>>> REL_16_9
#, c-format
msgid "definition of service \"%s\" not found"
msgstr "определение службы \"%s\" не найдено"

<<<<<<< HEAD
#: fe-connect.c:5119 fe-connect.c:5611
msgid "unterminated quoted string in connection info string\n"
msgstr "в строке соединения не хватает закрывающей кавычки\n"

#: fe-connect.c:5200
=======
#: fe-connect.c:5662
>>>>>>> REL_16_9
#, c-format
msgid "service file \"%s\" not found"
msgstr "файл определений служб \"%s\" не найден"

<<<<<<< HEAD
#: fe-connect.c:5226
=======
#: fe-connect.c:5675
>>>>>>> REL_16_9
#, c-format
msgid "line %d too long in service file \"%s\""
msgstr "слишком длинная строка (%d) в файле определений служб \"%s\""

<<<<<<< HEAD
#: fe-connect.c:5240
=======
#: fe-connect.c:5746 fe-connect.c:5789
>>>>>>> REL_16_9
#, c-format
msgid "syntax error in service file \"%s\", line %d"
msgstr "синтаксическая ошибка в файле определения служб \"%s\" (строка %d)"

<<<<<<< HEAD
#: fe-connect.c:5311 fe-connect.c:5355
#, c-format
msgid "syntax error in service file \"%s\", line %d\n"
msgstr "синтаксическая ошибка в файле определения служб \"%s\" (строка %d)\n"

#: fe-connect.c:5322
=======
#: fe-connect.c:5757
>>>>>>> REL_16_9
#, c-format
msgid ""
"nested service specifications not supported in service file \"%s\", line %d"
msgstr ""
"рекурсивные определения служб не поддерживаются (файл определения служб "
"\"%s\", строка %d)"

<<<<<<< HEAD
#: fe-connect.c:6043
=======
#: fe-connect.c:6496
>>>>>>> REL_16_9
#, c-format
msgid "invalid URI propagated to internal parser routine: \"%s\""
msgstr "во внутреннюю процедуру разбора строки передан ошибочный URI: \"%s\""

<<<<<<< HEAD
#: fe-connect.c:6120
=======
#: fe-connect.c:6573
>>>>>>> REL_16_9
#, c-format
msgid ""
"end of string reached when looking for matching \"]\" in IPv6 host address "
"in URI: \"%s\""
msgstr "URI не содержит символ \"]\" после адреса IPv6: \"%s\""

<<<<<<< HEAD
#: fe-connect.c:6127
=======
#: fe-connect.c:6580
>>>>>>> REL_16_9
#, c-format
msgid "IPv6 host address may not be empty in URI: \"%s\""
msgstr "IPv6, содержащийся в URI, не может быть пустым: \"%s\""

<<<<<<< HEAD
#: fe-connect.c:6142
=======
#: fe-connect.c:6595
>>>>>>> REL_16_9
#, c-format
msgid ""
"unexpected character \"%c\" at position %d in URI (expected \":\" or \"/\"): "
"\"%s\""
msgstr ""
"неожиданный символ \"%c\" в позиции %d в URI (ожидалось \":\" или \"/\"): "
"\"%s\""

<<<<<<< HEAD
#: fe-connect.c:6272
=======
#: fe-connect.c:6724
>>>>>>> REL_16_9
#, c-format
msgid "extra key/value separator \"=\" in URI query parameter: \"%s\""
msgstr "лишний разделитель ключа/значения \"=\" в параметрах URI: \"%s\""

<<<<<<< HEAD
#: fe-connect.c:6292
=======
#: fe-connect.c:6744
>>>>>>> REL_16_9
#, c-format
msgid "missing key/value separator \"=\" in URI query parameter: \"%s\""
msgstr "в параметрах URI не хватает разделителя ключа/значения \"=\": \"%s\""

<<<<<<< HEAD
#: fe-connect.c:6344
=======
#: fe-connect.c:6796
>>>>>>> REL_16_9
#, c-format
msgid "invalid URI query parameter: \"%s\""
msgstr "неверный параметр в URI: \"%s\""

<<<<<<< HEAD
#: fe-connect.c:6418
=======
#: fe-connect.c:6870
>>>>>>> REL_16_9
#, c-format
msgid "invalid percent-encoded token: \"%s\""
msgstr "неверный символ, закодированный с %%: \"%s\""

<<<<<<< HEAD
#: fe-connect.c:6428
=======
#: fe-connect.c:6880
>>>>>>> REL_16_9
#, c-format
msgid "forbidden value %%00 in percent-encoded value: \"%s\""
msgstr "недопустимое значение %%00 для символа, закодированного с %%: \"%s\""

<<<<<<< HEAD
#: fe-connect.c:6798
msgid "connection pointer is NULL\n"
msgstr "нулевой указатель соединения\n"

#: fe-connect.c:7086
=======
#: fe-connect.c:7244
msgid "connection pointer is NULL\n"
msgstr "нулевой указатель соединения\n"

#: fe-connect.c:7252 fe-exec.c:710 fe-exec.c:972 fe-exec.c:3321
#: fe-protocol3.c:969 fe-protocol3.c:1002
msgid "out of memory\n"
msgstr "нехватка памяти\n"

#: fe-connect.c:7543
>>>>>>> REL_16_9
#, c-format
msgid "WARNING: password file \"%s\" is not a plain file\n"
msgstr "ПРЕДУПРЕЖДЕНИЕ: файл паролей \"%s\" - не обычный файл\n"

<<<<<<< HEAD
#: fe-connect.c:7095
=======
#: fe-connect.c:7552
>>>>>>> REL_16_9
#, c-format
msgid ""
"WARNING: password file \"%s\" has group or world access; permissions should "
"be u=rw (0600) or less\n"
msgstr ""
"ПРЕДУПРЕЖДЕНИЕ: к файлу паролей \"%s\" имеют доступ все или группа; права "
"должны быть u=rw (0600) или более ограниченные\n"

<<<<<<< HEAD
#: fe-connect.c:7203
=======
#: fe-connect.c:7659
>>>>>>> REL_16_9
#, c-format
msgid "password retrieved from file \"%s\""
msgstr "пароль получен из файла \"%s\""

<<<<<<< HEAD
#: fe-exec.c:449 fe-exec.c:3304
=======
#: fe-exec.c:466 fe-exec.c:3395
>>>>>>> REL_16_9
#, c-format
msgid "row number %d is out of range 0..%d"
msgstr "номер записи %d вне диапазона 0..%d"

<<<<<<< HEAD
#: fe-exec.c:510 fe-protocol3.c:219 fe-protocol3.c:244 fe-protocol3.c:273
#: fe-protocol3.c:291 fe-protocol3.c:371 fe-protocol3.c:743
msgid "out of memory"
msgstr "нехватка памяти"

#: fe-exec.c:511 fe-protocol3.c:1939
=======
#: fe-exec.c:528 fe-protocol3.c:1971
>>>>>>> REL_16_9
#, c-format
msgid "%s"
msgstr "%s"

<<<<<<< HEAD
#: fe-exec.c:792
msgid "write to server failed\n"
msgstr "ошибка при передаче данных серверу\n"

#: fe-exec.c:864
msgid "NOTICE"
msgstr "ЗАМЕЧАНИЕ"

#: fe-exec.c:922
msgid "PGresult cannot support more than INT_MAX tuples"
msgstr "PGresult не может вместить больше чем INT_MAX кортежей"

#: fe-exec.c:934
msgid "size_t overflow"
msgstr "переполнение size_t"

#: fe-exec.c:1349 fe-exec.c:1454 fe-exec.c:1503
msgid "command string is a null pointer\n"
msgstr "указатель на командную строку нулевой\n"

#: fe-exec.c:1460 fe-exec.c:1509 fe-exec.c:1605
#, c-format
msgid "number of parameters must be between 0 and %d\n"
msgstr "число параметров должно быть от 0 до %d\n"

#: fe-exec.c:1497 fe-exec.c:1599
msgid "statement name is a null pointer\n"
msgstr "указатель на имя оператора нулевой\n"

#: fe-exec.c:1641 fe-exec.c:3157
msgid "no connection to the server\n"
msgstr "нет соединения с сервером\n"

#: fe-exec.c:1650 fe-exec.c:3166
msgid "another command is already in progress\n"
msgstr "уже выполняется другая команда\n"

#: fe-exec.c:1679
msgid "cannot queue commands during COPY\n"
msgstr "во время COPY нельзя добавлять команды в очередь\n"

#: fe-exec.c:1797
msgid "length must be given for binary parameter\n"
msgstr "для двоичного параметра должна быть указана длина\n"

#: fe-exec.c:2121
=======
#: fe-exec.c:831
#, c-format
msgid "write to server failed"
msgstr "ошибка при передаче данных серверу"

#: fe-exec.c:871
#, c-format
msgid "no error text available"
msgstr "текст ошибки отсутствует"

#: fe-exec.c:960
msgid "NOTICE"
msgstr "ЗАМЕЧАНИЕ"

#: fe-exec.c:1018
msgid "PGresult cannot support more than INT_MAX tuples"
msgstr "PGresult не может вместить больше чем INT_MAX кортежей"

#: fe-exec.c:1030
msgid "size_t overflow"
msgstr "переполнение size_t"

#: fe-exec.c:1446 fe-exec.c:1515 fe-exec.c:1561
>>>>>>> REL_16_9
#, c-format
msgid "command string is a null pointer"
msgstr "указатель на командную строку нулевой"

<<<<<<< HEAD
#: fe-exec.c:2141
=======
#: fe-exec.c:1452 fe-exec.c:2883
>>>>>>> REL_16_9
#, c-format
msgid "%s not allowed in pipeline mode"
msgstr "%s не допускается в конвейерном режиме"

<<<<<<< HEAD
#: fe-exec.c:2289
msgid ""
"synchronous command execution functions are not allowed in pipeline mode\n"
msgstr ""
"функции синхронного выполнения команд не допускаются в конвейерном режиме\n"

#: fe-exec.c:2311
msgid "COPY terminated by new PQexec"
msgstr "операция COPY прервана вызовом PQexec"

#: fe-exec.c:2328
msgid "PQexec not allowed during COPY BOTH\n"
msgstr "вызов PQexec не допускается в процессе COPY BOTH\n"

#: fe-exec.c:2556 fe-exec.c:2612 fe-exec.c:2681 fe-protocol3.c:1870
msgid "no COPY in progress\n"
msgstr "операция COPY не выполняется\n"

#: fe-exec.c:2858
msgid "PQfn not allowed in pipeline mode\n"
msgstr "PQfn не допускается в конвейерном режиме\n"

#: fe-exec.c:2866
msgid "connection in wrong state\n"
msgstr "соединение в неправильном состоянии\n"

#: fe-exec.c:2910
msgid "cannot enter pipeline mode, connection not idle\n"
msgstr "перейти в конвейерный режиме нельзя, соединение не простаивает\n"

#: fe-exec.c:2944 fe-exec.c:2961
msgid "cannot exit pipeline mode with uncollected results\n"
msgstr "выйти из конвейерного режима нельзя, не собрав все результаты\n"

#: fe-exec.c:2949
msgid "cannot exit pipeline mode while busy\n"
msgstr "выйти из конвейерного режима в занятом состоянии нельзя\n"

#: fe-exec.c:3091
msgid "cannot send pipeline when not in pipeline mode\n"
msgstr "отправить конвейер, не перейдя в конвейерный режим, нельзя\n"

#: fe-exec.c:3193
msgid "invalid ExecStatusType code"
msgstr "неверный код ExecStatusType"

#: fe-exec.c:3220
msgid "PGresult is not an error result\n"
msgstr "В PGresult не передан результат ошибки\n"

#: fe-exec.c:3288 fe-exec.c:3311
=======
#: fe-exec.c:1520 fe-exec.c:1566 fe-exec.c:1660
#, c-format
msgid "number of parameters must be between 0 and %d"
msgstr "число параметров должно быть от 0 до %d"

#: fe-exec.c:1556 fe-exec.c:1655
#, c-format
msgid "statement name is a null pointer"
msgstr "указатель на имя оператора нулевой"

#: fe-exec.c:1697 fe-exec.c:3241
#, c-format
msgid "no connection to the server"
msgstr "нет соединения с сервером"

#: fe-exec.c:1705 fe-exec.c:3249
#, c-format
msgid "another command is already in progress"
msgstr "уже выполняется другая команда"

#: fe-exec.c:1735
#, c-format
msgid "cannot queue commands during COPY"
msgstr "во время COPY нельзя добавлять команды в очередь"

#: fe-exec.c:1852
#, c-format
msgid "length must be given for binary parameter"
msgstr "для двоичного параметра должна быть указана длина"

#: fe-exec.c:2166
#, c-format
msgid "unexpected asyncStatus: %d"
msgstr "неожиданный asyncStatus: %d"

#: fe-exec.c:2322
#, c-format
msgid ""
"synchronous command execution functions are not allowed in pipeline mode"
msgstr ""
"функции синхронного выполнения команд не допускаются в конвейерном режиме"

#: fe-exec.c:2339
msgid "COPY terminated by new PQexec"
msgstr "операция COPY прервана вызовом PQexec"

#: fe-exec.c:2355
#, c-format
msgid "PQexec not allowed during COPY BOTH"
msgstr "вызов PQexec не допускается в процессе COPY BOTH"

#: fe-exec.c:2581 fe-exec.c:2636 fe-exec.c:2704 fe-protocol3.c:1902
#, c-format
msgid "no COPY in progress"
msgstr "операция COPY не выполняется"

#: fe-exec.c:2890
#, c-format
msgid "connection in wrong state"
msgstr "соединение в неправильном состоянии"

#: fe-exec.c:2933
#, c-format
msgid "cannot enter pipeline mode, connection not idle"
msgstr "перейти в конвейерный режиме нельзя, соединение не простаивает"

#: fe-exec.c:2969 fe-exec.c:2990
#, c-format
msgid "cannot exit pipeline mode with uncollected results"
msgstr "выйти из конвейерного режима нельзя, не собрав все результаты"

#: fe-exec.c:2973
#, c-format
msgid "cannot exit pipeline mode while busy"
msgstr "выйти из конвейерного режима в занятом состоянии нельзя"

#: fe-exec.c:2984
#, c-format
msgid "cannot exit pipeline mode while in COPY"
msgstr "выйти из конвейерного режима во время COPY нельзя"

#: fe-exec.c:3175
#, c-format
msgid "cannot send pipeline when not in pipeline mode"
msgstr "отправить конвейер, не перейдя в конвейерный режим, нельзя"

#: fe-exec.c:3284
msgid "invalid ExecStatusType code"
msgstr "неверный код ExecStatusType"

#: fe-exec.c:3311
msgid "PGresult is not an error result\n"
msgstr "В PGresult не передан результат ошибки\n"

#: fe-exec.c:3379 fe-exec.c:3402
>>>>>>> REL_16_9
#, c-format
msgid "column number %d is out of range 0..%d"
msgstr "номер столбца %d вне диапазона 0..%d"

<<<<<<< HEAD
#: fe-exec.c:3326
=======
#: fe-exec.c:3417
>>>>>>> REL_16_9
#, c-format
msgid "parameter number %d is out of range 0..%d"
msgstr "номер параметра %d вне диапазона 0..%d"

<<<<<<< HEAD
#: fe-exec.c:3636
=======
#: fe-exec.c:3728
>>>>>>> REL_16_9
#, c-format
msgid "could not interpret result from server: %s"
msgstr "не удалось интерпретировать ответ сервера: %s"

<<<<<<< HEAD
#: fe-exec.c:3896 fe-exec.c:3985
msgid "incomplete multibyte character\n"
msgstr "неполный многобайтный символ\n"
=======
#: fe-exec.c:4006 fe-exec.c:4119
#, c-format
msgid "incomplete multibyte character"
msgstr "неполный многобайтный символ"
>>>>>>> REL_16_9

#: fe-exec.c:4008 fe-exec.c:4138
#, c-format
msgid "invalid multibyte character"
msgstr "неверный многобайтный символ"

#: fe-gssapi-common.c:122
msgid "GSSAPI name import error"
msgstr "ошибка импорта имени в GSSAPI"

<<<<<<< HEAD
#: fe-lobj.c:145 fe-lobj.c:210 fe-lobj.c:403 fe-lobj.c:494 fe-lobj.c:568
#: fe-lobj.c:969 fe-lobj.c:977 fe-lobj.c:985 fe-lobj.c:993 fe-lobj.c:1001
#: fe-lobj.c:1009 fe-lobj.c:1017 fe-lobj.c:1025
#, c-format
msgid "cannot determine OID of function %s\n"
msgstr "определить OID функции %s нельзя\n"

#: fe-lobj.c:162
msgid "argument of lo_truncate exceeds integer range\n"
msgstr "аргумент lo_truncate не умещается в обычном целом\n"

#: fe-lobj.c:266
msgid "argument of lo_read exceeds integer range\n"
msgstr "аргумент lo_read не умещается в обычном целом\n"

#: fe-lobj.c:318
msgid "argument of lo_write exceeds integer range\n"
msgstr "аргумент lo_write не умещается в обычном целом\n"

#: fe-lobj.c:678 fe-lobj.c:789
=======
#: fe-lobj.c:144 fe-lobj.c:207 fe-lobj.c:397 fe-lobj.c:487 fe-lobj.c:560
#: fe-lobj.c:956 fe-lobj.c:963 fe-lobj.c:970 fe-lobj.c:977 fe-lobj.c:984
#: fe-lobj.c:991 fe-lobj.c:998 fe-lobj.c:1005
>>>>>>> REL_16_9
#, c-format
msgid "cannot determine OID of function %s"
msgstr "определить OID функции %s нельзя"

<<<<<<< HEAD
#: fe-lobj.c:734
=======
#: fe-lobj.c:160
>>>>>>> REL_16_9
#, c-format
msgid "argument of lo_truncate exceeds integer range"
msgstr "аргумент lo_truncate не умещается в обычном целом"

<<<<<<< HEAD
#: fe-lobj.c:810 fe-lobj.c:834
=======
#: fe-lobj.c:262
>>>>>>> REL_16_9
#, c-format
msgid "argument of lo_read exceeds integer range"
msgstr "аргумент lo_read не умещается в обычном целом"

<<<<<<< HEAD
#: fe-lobj.c:920
msgid "query to initialize large object functions did not return data\n"
msgstr "запрос инициализации функций для больших объектов не вернул данные\n"

#: fe-misc.c:242
=======
#: fe-lobj.c:313
#, c-format
msgid "argument of lo_write exceeds integer range"
msgstr "аргумент lo_write не умещается в обычном целом"

#: fe-lobj.c:669 fe-lobj.c:780
#, c-format
msgid "could not open file \"%s\": %s"
msgstr "не удалось открыть файл \"%s\": %s"

#: fe-lobj.c:725
#, c-format
msgid "could not read from file \"%s\": %s"
msgstr "не удалось прочитать файл \"%s\": %s"

#: fe-lobj.c:801 fe-lobj.c:824
#, c-format
msgid "could not write to file \"%s\": %s"
msgstr "не удалось записать файл \"%s\": %s"

#: fe-lobj.c:908
#, c-format
msgid "query to initialize large object functions did not return data"
msgstr "запрос инициализации функций для больших объектов не вернул данные"

#: fe-misc.c:240
>>>>>>> REL_16_9
#, c-format
msgid "integer of size %lu not supported by pqGetInt"
msgstr "функция pqGetInt не поддерживает integer размером %lu байт"

<<<<<<< HEAD
#: fe-misc.c:275
=======
#: fe-misc.c:273
>>>>>>> REL_16_9
#, c-format
msgid "integer of size %lu not supported by pqPutInt"
msgstr "функция pqPutInt не поддерживает integer размером %lu байт"

<<<<<<< HEAD
#: fe-misc.c:576 fe-misc.c:822
msgid "connection not open\n"
msgstr "соединение не открыто\n"

#: fe-misc.c:755 fe-secure-openssl.c:209 fe-secure-openssl.c:316
#: fe-secure.c:260 fe-secure.c:373
=======
#: fe-misc.c:573
#, c-format
msgid "connection not open"
msgstr "соединение не открыто"

#: fe-misc.c:751 fe-secure-openssl.c:210 fe-secure-openssl.c:316
#: fe-secure.c:259 fe-secure.c:426
#, c-format
>>>>>>> REL_16_9
msgid ""
"server closed the connection unexpectedly\n"
"\tThis probably means the server terminated abnormally\n"
"\tbefore or while processing the request."
msgstr ""
"сервер неожиданно закрыл соединение\n"
"\tСкорее всего сервер прекратил работу из-за сбоя\n"
"\tдо или в процессе выполнения запроса."

<<<<<<< HEAD
#: fe-misc.c:1015
msgid "timeout expired\n"
msgstr "тайм-аут\n"

#: fe-misc.c:1060
msgid "invalid socket\n"
msgstr "неверный сокет\n"

#: fe-misc.c:1083
#, c-format
msgid "%s() failed: %s\n"
msgstr "ошибка в %s(): %s\n"

#: fe-protocol3.c:196
=======
#: fe-misc.c:818
msgid "connection not open\n"
msgstr "соединение не открыто\n"

#: fe-misc.c:1003
#, c-format
msgid "timeout expired"
msgstr "тайм-аут"

#: fe-misc.c:1047
#, c-format
msgid "invalid socket"
msgstr "неверный сокет"

#: fe-misc.c:1069
#, c-format
msgid "%s() failed: %s"
msgstr "ошибка в %s(): %s"

#: fe-protocol3.c:182
>>>>>>> REL_16_9
#, c-format
msgid "message type 0x%02x arrived from server while idle"
msgstr "от сервера во время простоя получено сообщение типа 0x%02x"

<<<<<<< HEAD
#: fe-protocol3.c:403
msgid ""
"server sent data (\"D\" message) without prior row description (\"T\" "
"message)\n"
msgstr ""
"сервер отправил данные (сообщение \"D\") без предварительного описания "
"строки (сообщение \"T\")\n"

#: fe-protocol3.c:446
#, c-format
msgid "unexpected response from server; first received character was \"%c\"\n"
msgstr "неожиданный ответ сервера; первый полученный символ: \"%c\"\n"

#: fe-protocol3.c:471
#, c-format
msgid "message contents do not agree with length in message type \"%c\"\n"
msgstr "содержимое не соответствует длине в сообщении типа \"%c\"\n"

#: fe-protocol3.c:491
#, c-format
msgid "lost synchronization with server: got message type \"%c\", length %d\n"
msgstr ""
"потеряна синхронизация с сервером: получено сообщение типа \"%c\", длина %d\n"

#: fe-protocol3.c:543 fe-protocol3.c:583
msgid "insufficient data in \"T\" message"
msgstr "недостаточно данных в сообщении \"T\""

#: fe-protocol3.c:654 fe-protocol3.c:860
msgid "out of memory for query result"
msgstr "недостаточно памяти для результата запроса"

#: fe-protocol3.c:723
msgid "insufficient data in \"t\" message"
msgstr "недостаточно данных в сообщении \"t\""

#: fe-protocol3.c:782 fe-protocol3.c:814 fe-protocol3.c:832
msgid "insufficient data in \"D\" message"
msgstr "недостаточно данных в сообщении \"D\""

#: fe-protocol3.c:788
msgid "unexpected field count in \"D\" message"
msgstr "неверное число полей в сообщении \"D\""

#: fe-protocol3.c:1036
=======
#: fe-protocol3.c:380
#, c-format
msgid ""
"server sent data (\"D\" message) without prior row description (\"T\" "
"message)"
msgstr ""
"сервер отправил данные (сообщение \"D\") без предварительного описания "
"строки (сообщение \"T\")"

#: fe-protocol3.c:422
#, c-format
msgid "unexpected response from server; first received character was \"%c\""
msgstr "неожиданный ответ сервера; первый полученный символ: \"%c\""

#: fe-protocol3.c:445
#, c-format
msgid "message contents do not agree with length in message type \"%c\""
msgstr "содержимое не соответствует длине в сообщении типа \"%c\""

#: fe-protocol3.c:463
#, c-format
msgid "lost synchronization with server: got message type \"%c\", length %d"
msgstr ""
"потеряна синхронизация с сервером: получено сообщение типа \"%c\", длина %d"

#: fe-protocol3.c:515 fe-protocol3.c:555
msgid "insufficient data in \"T\" message"
msgstr "недостаточно данных в сообщении \"T\""

#: fe-protocol3.c:626 fe-protocol3.c:832
msgid "out of memory for query result"
msgstr "недостаточно памяти для результата запроса"

#: fe-protocol3.c:695
msgid "insufficient data in \"t\" message"
msgstr "недостаточно данных в сообщении \"t\""

#: fe-protocol3.c:754 fe-protocol3.c:786 fe-protocol3.c:804
msgid "insufficient data in \"D\" message"
msgstr "недостаточно данных в сообщении \"D\""

#: fe-protocol3.c:760
msgid "unexpected field count in \"D\" message"
msgstr "неверное число полей в сообщении \"D\""

#: fe-protocol3.c:1015
>>>>>>> REL_16_9
msgid "no error message available\n"
msgstr "нет сообщения об ошибке\n"

#. translator: %s represents a digit string
<<<<<<< HEAD
#: fe-protocol3.c:1084 fe-protocol3.c:1103
=======
#: fe-protocol3.c:1063 fe-protocol3.c:1082
>>>>>>> REL_16_9
#, c-format
msgid " at character %s"
msgstr " символ %s"

<<<<<<< HEAD
#: fe-protocol3.c:1116
=======
#: fe-protocol3.c:1095
>>>>>>> REL_16_9
#, c-format
msgid "DETAIL:  %s\n"
msgstr "ПОДРОБНОСТИ:  %s\n"

<<<<<<< HEAD
#: fe-protocol3.c:1119
=======
#: fe-protocol3.c:1098
>>>>>>> REL_16_9
#, c-format
msgid "HINT:  %s\n"
msgstr "ПОДСКАЗКА:  %s\n"

<<<<<<< HEAD
#: fe-protocol3.c:1122
=======
#: fe-protocol3.c:1101
>>>>>>> REL_16_9
#, c-format
msgid "QUERY:  %s\n"
msgstr "ЗАПРОС:  %s\n"

<<<<<<< HEAD
#: fe-protocol3.c:1129
=======
#: fe-protocol3.c:1108
>>>>>>> REL_16_9
#, c-format
msgid "CONTEXT:  %s\n"
msgstr "КОНТЕКСТ:  %s\n"

<<<<<<< HEAD
#: fe-protocol3.c:1138
=======
#: fe-protocol3.c:1117
>>>>>>> REL_16_9
#, c-format
msgid "SCHEMA NAME:  %s\n"
msgstr "СХЕМА:  %s\n"

<<<<<<< HEAD
#: fe-protocol3.c:1142
=======
#: fe-protocol3.c:1121
>>>>>>> REL_16_9
#, c-format
msgid "TABLE NAME:  %s\n"
msgstr "ТАБЛИЦА:  %s\n"

<<<<<<< HEAD
#: fe-protocol3.c:1146
=======
#: fe-protocol3.c:1125
>>>>>>> REL_16_9
#, c-format
msgid "COLUMN NAME:  %s\n"
msgstr "СТОЛБЕЦ:  %s\n"

<<<<<<< HEAD
#: fe-protocol3.c:1150
=======
#: fe-protocol3.c:1129
>>>>>>> REL_16_9
#, c-format
msgid "DATATYPE NAME:  %s\n"
msgstr "ТИП ДАННЫХ:  %s\n"

<<<<<<< HEAD
#: fe-protocol3.c:1154
=======
#: fe-protocol3.c:1133
>>>>>>> REL_16_9
#, c-format
msgid "CONSTRAINT NAME:  %s\n"
msgstr "ОГРАНИЧЕНИЕ:  %s\n"

<<<<<<< HEAD
#: fe-protocol3.c:1166
msgid "LOCATION:  "
msgstr "ПОЛОЖЕНИЕ:  "

#: fe-protocol3.c:1168
=======
#: fe-protocol3.c:1145
msgid "LOCATION:  "
msgstr "ПОЛОЖЕНИЕ:  "

#: fe-protocol3.c:1147
>>>>>>> REL_16_9
#, c-format
msgid "%s, "
msgstr "%s, "

<<<<<<< HEAD
#: fe-protocol3.c:1170
=======
#: fe-protocol3.c:1149
>>>>>>> REL_16_9
#, c-format
msgid "%s:%s"
msgstr "%s:%s"

<<<<<<< HEAD
#: fe-protocol3.c:1365
=======
#: fe-protocol3.c:1344
>>>>>>> REL_16_9
#, c-format
msgid "LINE %d: "
msgstr "СТРОКА %d: "

<<<<<<< HEAD
#: fe-protocol3.c:1764
msgid "PQgetline: not doing text COPY OUT\n"
msgstr "PQgetline можно вызывать только во время COPY OUT с текстом\n"

#: fe-protocol3.c:2130
#, c-format
msgid "protocol error: id=0x%x\n"
msgstr "ошибка протокола: id=0x%x\n"

#: fe-secure-common.c:124
msgid "SSL certificate's name contains embedded null\n"
msgstr "имя в SSL-сертификате включает нулевой байт\n"

#: fe-secure-common.c:171
msgid "host name must be specified for a verified SSL connection\n"
msgstr "для проверенного SSL-соединения требуется указать имя узла\n"

#: fe-secure-common.c:196
=======
#: fe-protocol3.c:1418
>>>>>>> REL_16_9
#, c-format
msgid ""
"protocol version not supported by server: client uses %u.%u, server supports "
"up to %u.%u"
msgstr ""
"сервер не поддерживает нужную версию протокола: клиент использует %u.%u, "
"сервер поддерживает версии до %u.%u"

#: fe-protocol3.c:1424
#, c-format
msgid "protocol extension not supported by server: %s"
msgid_plural "protocol extensions not supported by server: %s"
msgstr[0] "сервер не поддерживает это расширение протокола: %s"
msgstr[1] "сервер не поддерживает эти расширения протокола: %s"
msgstr[2] "сервер не поддерживает эти расширения протокола: %s"

#: fe-protocol3.c:1432
#, c-format
msgid "invalid %s message"
msgstr "неверное сообщение %s"

#: fe-protocol3.c:1797
#, c-format
msgid "PQgetline: not doing text COPY OUT"
msgstr "PQgetline можно вызывать только во время COPY OUT с текстом"

#: fe-protocol3.c:2171
#, c-format
msgid "protocol error: no function result"
msgstr "ошибка протокола: нет результата функции"

#: fe-protocol3.c:2182
#, c-format
msgid "protocol error: id=0x%x"
msgstr "ошибка протокола: id=0x%x"

#: fe-secure-common.c:123
#, c-format
msgid "SSL certificate's name contains embedded null"
msgstr "имя в SSL-сертификате включает нулевой байт"

#: fe-secure-common.c:228
#, c-format
msgid "certificate contains IP address with invalid length %zu"
msgstr "сертификат содержит IP-адрес неверной длины %zu"

#: fe-secure-common.c:237
#, c-format
msgid "could not convert certificate's IP address to string: %s"
msgstr "не удалось преобразовать в строку IP-адрес из сертификата: %s"

#: fe-secure-common.c:269
#, c-format
msgid "host name must be specified for a verified SSL connection"
msgstr "для проверенного SSL-соединения должно указываться имя узла"

#: fe-secure-common.c:286
#, c-format
msgid ""
"server certificate for \"%s\" (and %d other name) does not match host name "
"\"%s\""
msgid_plural ""
"server certificate for \"%s\" (and %d other names) does not match host name "
"\"%s\""
msgstr[0] ""
"серверный сертификат для \"%s\" (и %d другого имени) не соответствует имени "
"сервера \"%s\""
msgstr[1] ""
"серверный сертификат для \"%s\" (и %d других имён) не соответствует имени "
"сервера \"%s\""
msgstr[2] ""
"серверный сертификат для \"%s\" (и %d других имён) не соответствует имени "
"сервера \"%s\""

#: fe-secure-common.c:294
#, c-format
msgid "server certificate for \"%s\" does not match host name \"%s\""
msgstr "серверный сертификат для \"%s\" не соответствует имени сервера \"%s\""

#: fe-secure-common.c:299
#, c-format
msgid "could not get server's host name from server certificate"
msgstr "не удалось получить имя сервера из серверного сертификата"

#: fe-secure-gssapi.c:194
msgid "GSSAPI wrap error"
msgstr "ошибка обёртывания сообщения в GSSAPI"

#: fe-secure-gssapi.c:201
#, c-format
msgid "outgoing GSSAPI message would not use confidentiality"
msgstr "исходящее сообщение GSSAPI не будет защищено"

#: fe-secure-gssapi.c:208
#, c-format
msgid "client tried to send oversize GSSAPI packet (%zu > %zu)"
msgstr "клиент попытался передать чрезмерно большой пакет GSSAPI (%zu > %zu)"

#: fe-secure-gssapi.c:347 fe-secure-gssapi.c:589
#, c-format
msgid "oversize GSSAPI packet sent by the server (%zu > %zu)"
msgstr "сервер передал чрезмерно большой пакет GSSAPI (%zu > %zu)"

#: fe-secure-gssapi.c:386
msgid "GSSAPI unwrap error"
msgstr "ошибка развёртывания сообщения в GSSAPI"

#: fe-secure-gssapi.c:395
#, c-format
msgid "incoming GSSAPI message did not use confidentiality"
msgstr "входящее сообщение GSSAPI не защищено"

#: fe-secure-gssapi.c:652
msgid "could not initiate GSSAPI security context"
msgstr "не удалось инициализировать контекст безопасности GSSAPI"

#: fe-secure-gssapi.c:681
msgid "GSSAPI size check error"
msgstr "ошибка проверки размера в GSSAPI"

#: fe-secure-gssapi.c:692
msgid "GSSAPI context establishment error"
msgstr "ошибка установления контекста в GSSAPI"

<<<<<<< HEAD
#: fe-secure-openssl.c:214 fe-secure-openssl.c:321 fe-secure-openssl.c:1367
=======
#: fe-secure-openssl.c:214 fe-secure-openssl.c:320 fe-secure-openssl.c:1518
>>>>>>> REL_16_9
#, c-format
msgid "SSL SYSCALL error: %s"
msgstr "ошибка SSL SYSCALL: %s"

<<<<<<< HEAD
#: fe-secure-openssl.c:221 fe-secure-openssl.c:328 fe-secure-openssl.c:1371
msgid "SSL SYSCALL error: EOF detected\n"
msgstr "ошибка SSL SYSCALL: конец файла (EOF)\n"

#: fe-secure-openssl.c:232 fe-secure-openssl.c:339 fe-secure-openssl.c:1380
=======
#: fe-secure-openssl.c:220 fe-secure-openssl.c:326 fe-secure-openssl.c:1521
>>>>>>> REL_16_9
#, c-format
msgid "SSL SYSCALL error: EOF detected"
msgstr "ошибка SSL SYSCALL: конец файла (EOF)"

<<<<<<< HEAD
#: fe-secure-openssl.c:247 fe-secure-openssl.c:354
msgid "SSL connection has been closed unexpectedly\n"
msgstr "SSL-соединение было неожиданно закрыто\n"

#: fe-secure-openssl.c:253 fe-secure-openssl.c:360 fe-secure-openssl.c:1430
=======
#: fe-secure-openssl.c:230 fe-secure-openssl.c:336 fe-secure-openssl.c:1529
>>>>>>> REL_16_9
#, c-format
msgid "SSL error: %s"
msgstr "ошибка SSL: %s"

#: fe-secure-openssl.c:244 fe-secure-openssl.c:350
#, c-format
msgid "SSL connection has been closed unexpectedly"
msgstr "SSL-соединение было неожиданно закрыто"

<<<<<<< HEAD
#: fe-secure-openssl.c:431
msgid "could not generate peer certificate hash\n"
msgstr "не удалось сгенерировать хеш сертификата сервера\n"

#: fe-secure-openssl.c:488
msgid "SSL certificate's name entry is missing\n"
msgstr "запись имени в SSL-сертификате отсутствует\n"

#: fe-secure-openssl.c:822
=======
#: fe-secure-openssl.c:249 fe-secure-openssl.c:355 fe-secure-openssl.c:1576
>>>>>>> REL_16_9
#, c-format
msgid "unrecognized SSL error code: %d"
msgstr "нераспознанный код ошибки SSL: %d"

<<<<<<< HEAD
#: fe-secure-openssl.c:861
=======
#: fe-secure-openssl.c:398
>>>>>>> REL_16_9
#, c-format
msgid "could not determine server certificate signature algorithm"
msgstr "не удалось определить алгоритм подписи сертификата сервера"

<<<<<<< HEAD
#: fe-secure-openssl.c:872
=======
#: fe-secure-openssl.c:418
>>>>>>> REL_16_9
#, c-format
msgid "could not find digest for NID %s"
msgstr "не удалось найти алгоритм хеширования по NID %s"

<<<<<<< HEAD
#: fe-secure-openssl.c:890
=======
#: fe-secure-openssl.c:427
>>>>>>> REL_16_9
#, c-format
msgid "could not generate peer certificate hash"
msgstr "не удалось сгенерировать хеш сертификата сервера"

<<<<<<< HEAD
#: fe-secure-openssl.c:901
=======
#: fe-secure-openssl.c:510
>>>>>>> REL_16_9
#, c-format
msgid "SSL certificate's name entry is missing"
msgstr "в SSL-сертификате отсутствует запись имени"

<<<<<<< HEAD
#: fe-secure-openssl.c:937
=======
#: fe-secure-openssl.c:544
>>>>>>> REL_16_9
#, c-format
msgid "SSL certificate's address entry is missing"
msgstr "в SSL-сертификате отсутствует запись адреса"

<<<<<<< HEAD
#: fe-secure-openssl.c:990
=======
#: fe-secure-openssl.c:945
#, c-format
msgid "could not create SSL context: %s"
msgstr "не удалось создать контекст SSL: %s"

#: fe-secure-openssl.c:987
#, c-format
msgid "invalid value \"%s\" for minimum SSL protocol version"
msgstr "неверное значение \"%s\" для минимальной версии протокола SSL"

#: fe-secure-openssl.c:997
#, c-format
msgid "could not set minimum SSL protocol version: %s"
msgstr "не удалось задать минимальную версию протокола SSL: %s"

#: fe-secure-openssl.c:1013
#, c-format
msgid "invalid value \"%s\" for maximum SSL protocol version"
msgstr "неверное значение \"%s\" для максимальной версии протокола SSL"

#: fe-secure-openssl.c:1023
#, c-format
msgid "could not set maximum SSL protocol version: %s"
msgstr "не удалось задать максимальную версию протокола SSL: %s"

#: fe-secure-openssl.c:1061
#, c-format
msgid "could not load system root certificate paths: %s"
msgstr "не удалось выбрать системные пути для корневых сертификатов: %s"

#: fe-secure-openssl.c:1078
#, c-format
msgid "could not read root certificate file \"%s\": %s"
msgstr "не удалось прочитать файл корневых сертификатов \"%s\": %s"

#: fe-secure-openssl.c:1130
#, c-format
>>>>>>> REL_16_9
msgid ""
"could not get home directory to locate root certificate file\n"
"Either provide the file, use the system's trusted roots with "
"sslrootcert=system, or change sslmode to disable server certificate "
"verification."
msgstr ""
"не удалось получить домашний каталог для поиска файла корневых сертификатов\n"
"Укажите полный путь к файлу, используйте системные доверенные сертификаты "
"(sslrootcert=system) или отключите проверку сертификата сервера, изменив "
"sslmode."

<<<<<<< HEAD
#: fe-secure-openssl.c:994
=======
#: fe-secure-openssl.c:1133
>>>>>>> REL_16_9
#, c-format
msgid ""
"root certificate file \"%s\" does not exist\n"
"Either provide the file, use the system's trusted roots with "
"sslrootcert=system, or change sslmode to disable server certificate "
"verification."
msgstr ""
"файл корневых сертификатов \"%s\" не существует\n"
"Укажите полный путь к файлу, используйте системные доверенные сертификаты "
"(sslrootcert=system) или отключите проверку сертификата сервера, изменив "
"sslmode."

<<<<<<< HEAD
#: fe-secure-openssl.c:1025
#, c-format
msgid "could not open certificate file \"%s\": %s\n"
msgstr "не удалось открыть файл сертификата \"%s\": %s\n"

#: fe-secure-openssl.c:1044
#, c-format
msgid "could not read certificate file \"%s\": %s\n"
msgstr "не удалось прочитать файл сертификата \"%s\": %s\n"

#: fe-secure-openssl.c:1069
#, c-format
msgid "could not establish SSL connection: %s\n"
msgstr "не удалось установить SSL-соединение: %s\n"

#: fe-secure-openssl.c:1103
#, c-format
msgid "could not set SSL Server Name Indication (SNI): %s\n"
msgstr ""
"не удалось задать SNI (Server Name Indication) для SSL-подключения: %s\n"

#: fe-secure-openssl.c:1149
#, c-format
msgid "could not load SSL engine \"%s\": %s\n"
msgstr "не удалось загрузить модуль SSL ENGINE \"%s\": %s\n"

#: fe-secure-openssl.c:1161
#, c-format
msgid "could not initialize SSL engine \"%s\": %s\n"
msgstr "не удалось инициализировать модуль SSL ENGINE \"%s\": %s\n"

#: fe-secure-openssl.c:1177
#, c-format
msgid "could not read private SSL key \"%s\" from engine \"%s\": %s\n"
msgstr "не удалось прочитать закрытый ключ SSL \"%s\" из модуля \"%s\": %s\n"

#: fe-secure-openssl.c:1191
#, c-format
msgid "could not load private SSL key \"%s\" from engine \"%s\": %s\n"
msgstr "не удалось загрузить закрытый ключ SSL \"%s\" из модуля \"%s\": %s\n"
=======
#: fe-secure-openssl.c:1168
#, c-format
msgid "could not open certificate file \"%s\": %s"
msgstr "не удалось открыть файл сертификата \"%s\": %s"
>>>>>>> REL_16_9

#: fe-secure-openssl.c:1228
#, c-format
msgid "could not read certificate file \"%s\": %s"
msgstr "не удалось прочитать файл сертификата \"%s\": %s"

<<<<<<< HEAD
#: fe-secure-openssl.c:1237
#, c-format
msgid "private key file \"%s\" is not a regular file\n"
msgstr "файл закрытого ключа \"%s\" - не обычный файл\n"

#: fe-secure-openssl.c:1261
#, c-format
msgid "private key file \"%s\" must be owned by the current user or root\n"
msgstr ""
"файл закрытого ключа \"%s\" должен принадлежать root или текущему "
"пользователю\n"

#: fe-secure-openssl.c:1270
#, c-format
msgid ""
"private key file \"%s\" has group or world access; file must have "
"permissions u=rw (0600) or less if owned by the current user, or permissions "
"u=rw,g=r (0640) or less if owned by root\n"
msgstr ""
"к файлу закрытого ключа \"%s\" имеют доступ все или группа; для него должны "
"быть заданы разрешения u=rw (0600) или более строгие, если он принадлежит "
"текущему пользователю, либо u=rw,g=r (0640) или более строгие, если он "
"принадлежит root\n"

#: fe-secure-openssl.c:1295
=======
#: fe-secure-openssl.c:1210
#, c-format
msgid "could not establish SSL connection: %s"
msgstr "не удалось установить SSL-соединение: %s"

#: fe-secure-openssl.c:1242
>>>>>>> REL_16_9
#, c-format
msgid "could not set SSL Server Name Indication (SNI): %s"
msgstr "не удалось задать SNI (Server Name Indication) для SSL-подключения: %s"

<<<<<<< HEAD
#: fe-secure-openssl.c:1313
=======
#: fe-secure-openssl.c:1286
>>>>>>> REL_16_9
#, c-format
msgid "could not load SSL engine \"%s\": %s"
msgstr "не удалось загрузить модуль SSL ENGINE \"%s\": %s"

<<<<<<< HEAD
#: fe-secure-openssl.c:1413
=======
#: fe-secure-openssl.c:1297
>>>>>>> REL_16_9
#, c-format
msgid "could not initialize SSL engine \"%s\": %s"
msgstr "не удалось инициализировать модуль SSL ENGINE \"%s\": %s"

#: fe-secure-openssl.c:1312
#, c-format
msgid "could not read private SSL key \"%s\" from engine \"%s\": %s"
msgstr "не удалось прочитать закрытый ключ SSL \"%s\" из модуля \"%s\": %s"

#: fe-secure-openssl.c:1325
#, c-format
msgid "could not load private SSL key \"%s\" from engine \"%s\": %s"
msgstr "не удалось загрузить закрытый ключ SSL \"%s\" из модуля \"%s\": %s"

#: fe-secure-openssl.c:1362
#, c-format
msgid "certificate present, but not private key file \"%s\""
msgstr "при наличии сертификата отсутствует файл закрытого ключа \"%s\""

#: fe-secure-openssl.c:1365
#, c-format
msgid "could not stat private key file \"%s\": %m"
msgstr "не удалось получить информацию о файле закрытого ключа \"%s\": %m"

#: fe-secure-openssl.c:1449
#, c-format
msgid "private key file \"%s\" is not a regular file"
msgstr "файл закрытого ключа \"%s\" - не обычный файл"

<<<<<<< HEAD
#: fe-secure-openssl.c:1555
=======
#: fe-secure-openssl.c:1406
#, c-format
msgid ""
"private key file \"%s\" has group or world access; file must have "
"permissions u=rw (0600) or less if owned by the current user, or permissions "
"u=rw,g=r (0640) or less if owned by root"
msgstr ""
"к файлу закрытого ключа \"%s\" имеют доступ все или группа; для него должны "
"быть заданы разрешения u=rw (0600) или более строгие, если он принадлежит "
"текущему пользователю, либо u=rw,g=r (0640) или более строгие, если он "
"принадлежит root"

#: fe-secure-openssl.c:1430
#, c-format
msgid "could not load private key file \"%s\": %s"
msgstr "не удалось загрузить файл закрытого ключа \"%s\": %s"

#: fe-secure-openssl.c:1446
#, c-format
msgid "certificate does not match private key file \"%s\": %s"
msgstr "сертификат не соответствует файлу закрытого ключа \"%s\": %s"

#: fe-secure-openssl.c:1515
#, c-format
msgid "SSL error: certificate verify failed: %s"
msgstr "ошибка SSL: не удалось проверить сертификат: %s"

#: fe-secure-openssl.c:1560
#, c-format
msgid ""
"This may indicate that the server does not support any SSL protocol version "
"between %s and %s."
msgstr ""
"Это может указывать на то, что сервер не поддерживает ни одну версию "
"протокола SSL между %s и %s."

#: fe-secure-openssl.c:1593
#, c-format
msgid "certificate could not be obtained: %s"
msgstr "не удалось получить сертификат: %s"

#: fe-secure-openssl.c:1699
>>>>>>> REL_16_9
#, c-format
msgid "no SSL error reported"
msgstr "нет сообщения об ошибке SSL"

<<<<<<< HEAD
#: fe-secure-openssl.c:1564
=======
#: fe-secure-openssl.c:1725
>>>>>>> REL_16_9
#, c-format
msgid "SSL error code %lu"
msgstr "код ошибки SSL: %lu"

<<<<<<< HEAD
#: fe-secure-openssl.c:1812
=======
#: fe-secure-openssl.c:2015
>>>>>>> REL_16_9
#, c-format
msgid "WARNING: sslpassword truncated\n"
msgstr "ПРЕДУПРЕЖДЕНИЕ: значение sslpassword усечено\n"

<<<<<<< HEAD
#: fe-secure.c:267
=======
#: fe-secure.c:270
>>>>>>> REL_16_9
#, c-format
msgid "could not receive data from server: %s"
msgstr "не удалось получить данные с сервера: %s"

<<<<<<< HEAD
#: fe-secure.c:380
=======
#: fe-secure.c:441
>>>>>>> REL_16_9
#, c-format
msgid "could not send data to server: %s"
msgstr "не удалось передать данные серверу: %s"

#: win32.c:310
#, c-format
msgid "unrecognized socket error: 0x%08X/%d"
msgstr "нераспознанная ошибка сокета: 0x%08X/%d"

<<<<<<< HEAD
=======
#, c-format
#~ msgid "keepalives parameter must be an integer"
#~ msgstr "параметр keepalives должен быть целым числом"

#~ msgid "SCM_CRED authentication method not supported\n"
#~ msgstr "аутентификация SCM_CRED не поддерживается\n"

#~ msgid ""
#~ "server sent data (\"D\" message) without prior row description (\"T\" "
#~ "message)\n"
#~ msgstr ""
#~ "сервер отправил данные (сообщение \"D\") без предварительного описания "
#~ "строки (сообщение \"T\")\n"

#~ msgid "PGEventProc \"%s\" failed during PGEVT_CONNRESET event\n"
#~ msgstr "ошибка в PGEventProc \"%s\" при обработке события PGEVT_CONNRESET\n"

#~ msgid "PGEventProc \"%s\" failed during PGEVT_RESULTCREATE event\n"
#~ msgstr ""
#~ "ошибка в PGEventProc \"%s\" при обработке события PGEVT_RESULTCREATE\n"

>>>>>>> REL_16_9
#~ msgid "invalid channel_binding value: \"%s\"\n"
#~ msgstr "неверное значение channel_binding: \"%s\"\n"

#~ msgid "invalid ssl_min_protocol_version value: \"%s\"\n"
#~ msgstr "неверное значение ssl_min_protocol_version: \"%s\"\n"

#~ msgid "invalid ssl_max_protocol_version value: \"%s\"\n"
#~ msgstr "неверное значение ssl_max_protocol_version: \"%s\"\n"

#~ msgid "invalid gssencmode value: \"%s\"\n"
#~ msgstr "неверное значение gssencmode: \"%s\"\n"

#~ msgid "invalid target_session_attrs value: \"%s\"\n"
#~ msgstr "неверное значение target_session_attrs: \"%s\"\n"

#~ msgid ""
#~ "could not connect to server: %s\n"
#~ "\tIs the server running on host \"%s\" (%s) and accepting\n"
#~ "\tTCP/IP connections on port %s?\n"
#~ msgstr ""
#~ "не удалось подключиться к серверу: %s\n"
#~ "\tОн действительно работает по адресу \"%s\" (%s)\n"
#~ "\t и принимает TCP-соединения (порт %s)?\n"

#~ msgid "setsockopt(%s) failed: %s\n"
#~ msgstr "ошибка в setsockopt(%s): %s\n"

#~ msgid "WSAIoctl(SIO_KEEPALIVE_VALS) failed: %ui\n"
#~ msgstr "ошибка в WSAIoctl(SIO_KEEPALIVE_VALS): %ui\n"

#~ msgid "could not make a writable connection to server \"%s:%s\"\n"
#~ msgstr ""
#~ "не удалось установить подключение для чтения/записи к серверу \"%s:%s\"\n"

#~ msgid "test \"SHOW transaction_read_only\" failed on server \"%s:%s\"\n"
#~ msgstr ""
#~ "проверка \"SHOW transaction_read_only\" не пройдена на сервере \"%s:%s\"\n"

#~ msgid "function requires at least protocol version 3.0\n"
#~ msgstr "функция требует протокол минимум версии 3.0\n"

#~ msgid "COPY IN state must be terminated first\n"
#~ msgstr "сначала должно завершиться состояние COPY IN\n"

#~ msgid "COPY OUT state must be terminated first\n"
#~ msgstr "сначала должно завершиться состояние COPY OUT\n"

#~ msgid "cannot determine OID of function lo_truncate\n"
#~ msgstr "не удалось определить OID функции lo_truncate\n"

#~ msgid "cannot determine OID of function lo_truncate64\n"
#~ msgstr "не удалось определить OID функции lo_truncate64\n"

#~ msgid "cannot determine OID of function lo_lseek64\n"
#~ msgstr "не удалось определить OID функции lo_lseek64\n"

#~ msgid "cannot determine OID of function lo_create\n"
#~ msgstr "не удалось определить OID функции lo_create\n"

#~ msgid "cannot determine OID of function lo_tell64\n"
#~ msgstr "не удалось определить OID функции lo_tell64\n"

#~ msgid "cannot determine OID of function lo_open\n"
#~ msgstr "не удалось определить OID функции lo_open\n"

#~ msgid "cannot determine OID of function lo_creat\n"
#~ msgstr "не удалось определить OID функции lo_creat\n"

#~ msgid "cannot determine OID of function lo_unlink\n"
#~ msgstr "не удалось определить OID функции lo_unlink\n"

#~ msgid "cannot determine OID of function lo_lseek\n"
#~ msgstr "не удалось определить OID функции lo_lseek\n"

#~ msgid "cannot determine OID of function loread\n"
#~ msgstr "не удалось определить OID функции loread\n"

#~ msgid "cannot determine OID of function lowrite\n"
#~ msgstr "не удалось определить OID функции lowrite\n"

#~ msgid "invalid setenv state %c, probably indicative of memory corruption\n"
#~ msgstr "неверное состояние setenv %c - возможно разрушение памяти\n"

#~ msgid "invalid state %c, probably indicative of memory corruption\n"
#~ msgstr "неверное состояние %c - возможно разрушение памяти\n"

#~ msgid ""
#~ "unexpected character %c following empty query response (\"I\" message)"
#~ msgstr "неожиданный символ %c вслед за пустым ответом (сообщение \"I\")"

#~ msgid ""
<<<<<<< HEAD
#~ "server sent data (\"D\" message) without prior row description (\"T\" "
#~ "message)"
#~ msgstr ""
#~ "сервер отправил данные (сообщение \"D\") без предварительного описания "
#~ "строки (сообщение \"T\")"

#~ msgid ""
#~ "server sent binary data (\"B\" message) without prior row description (\"T"
#~ "\" message)"
=======
#~ "server sent binary data (\"B\" message) without prior row description "
#~ "(\"T\" message)"
>>>>>>> REL_16_9
#~ msgstr ""
#~ "сервер отправил двоичные данные (сообщение \"B\") без предварительного "
#~ "описания строки (сообщение \"T\")"

#~ msgid "lost synchronization with server, resetting connection"
#~ msgstr "потеряна синхронизация с сервером; попытка восстановить соединение"

#~ msgid "extraneous data in \"T\" message"
#~ msgstr "лишние данные в сообщении \"T\""

#~ msgid "extraneous data in \"t\" message"
#~ msgstr "лишние данные в сообщении \"t\""

#~ msgid "extraneous data in \"D\" message"
#~ msgstr "лишние данные в сообщении \"D\""

#~ msgid "SSL library does not support CRL certificates (file \"%s\")\n"
#~ msgstr "Библиотека SSL не поддерживает проверку CRL (файл \"%s\")\n"

#~ msgid "could not get home directory to locate password file\n"
#~ msgstr "не удалось получить домашний каталог для загрузки файла паролей\n"

#~ msgid "could not get home directory to locate service definition file"
#~ msgstr ""
#~ "не удалось получить домашний каталог для загрузки файла определений служб"

#~ msgid "setsockopt(TCP_KEEPIDLE) failed: %s\n"
#~ msgstr "ошибка в setsockopt(TCP_KEEPIDLE): %s\n"

#~ msgid "setsockopt(TCP_KEEPALIVE) failed: %s\n"
#~ msgstr "ошибка в setsockopt(TCP_KEEPALIVE): %s\n"

#~ msgid "setsockopt(TCP_KEEPINTVL) failed: %s\n"
#~ msgstr "ошибка в setsockopt(TCP_KEEPINTVL): %s\n"

#~ msgid "setsockopt(SO_KEEPALIVE) failed: %s\n"
#~ msgstr "ошибка в setsockopt(SO_KEEPALIVE): %s\n"

#~ msgid "could not acquire mutex: %s\n"
#~ msgstr "не удалось заблокировать семафор: %s\n"

#~ msgid "socket not open\n"
#~ msgstr "сокет не открыт\n"

#~ msgid "unrecognized return value from row processor"
#~ msgstr "процессор строк вернул нераспознанное значение"

#~ msgid "could not restore nonblocking mode on socket: %s\n"
#~ msgstr "не удалось вернуть сокет в неблокирующий режим: %s\n"

#~ msgid "Kerberos 5 authentication rejected: %*s\n"
#~ msgstr "аутентификация Kerberos 5 не пройдена: %*s\n"

#~ msgid "could not set socket to blocking mode: %s\n"
#~ msgstr "не удалось перевести сокет в блокирующий режим: %s\n"
